src/main/java/ru/javarush/quest/impl/AnswerImpl.java

```diff
@@ -0,0 +1,62 @@
+package ru.javarush.quest.impl;
+
+import ru.javarush.quest.entity.Answer;
+import ru.javarush.quest.service.AbstractDao;
+import ru.javarush.quest.service.AnswerDao;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class AnswerImpl extends AbstractDao implements AnswerDao {
+    @Override
+    public Answer getAnswersInOrder(int count) {
+
+        Answer answer = null;
+        try (Connection connection = connect();
+             Statement stmt = connection.createStatement()) {
+            stmt.execute("SELECT * from answers where id=" + count);
+            ResultSet resultSet = stmt.getResultSet();
```

Comment:
it seems that https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html implements Autoclosable

Reply:
ResultSet close method was added in finally block